server.js

```diff
@@ -8,7 +8,7 @@ if (!process.env.DEMO_DEPLOY) {
 }
 
 const server = require('./app');
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 5000;
```

Comment:
I see the value on building the assets in advance but what is the value of having 2 node processes running? I think it will make it harder for users since now they need to have 2 things running.